roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -87,10 +87,20 @@ data:
             }
 
             location / {
+                rewrite ^/$ /eda/ permanent; # Redirects all traffic from root to /eda/
```

Comment:
Why not use here `return 301 $scheme://$host/eda/;` as well

Reply:
Good point, I've updated the PR.  Thanks!

Reply:
ok, but one of both, or you do the rewrite or the redirect. `return 301 $scheme://$host/eda/;` I would say is simpler and efficient.

Reply:
Thank you for clarifying, I just updated it.  In testing, I am able to navigate to the UI and it redirects correctly.